src/Kros.EventBusDoc.Generator/BusentScour/XmlReader/BXmlReader.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Kros.EventBusDoc.Generator.BusentScour.XmlReader
+{
+    public class BXmlReader : IXmlReader
+    {
+        #region Properties
+
+        public XmlDocument Document { get; private set; }
+
+        #endregion
+
+        #region Member Functions
+
+        public string GetSummary(string item)
+        {
+            return null;
+        }
+
+        public void Load(string documentPath)
+        {
+            Document = new XmlDocument();
+            try
+            {
+                Document.Load(documentPath);
+            }
+            catch (Exception e)
+            {
+                throw e; // so far to get notify
+            }
+        }
+
+        public string GetSummary(Type type, PropertyInfo property = null)
```

Comment:
Používame Expression-bodied members:
```suggestion
        public string GetSummary(Type type, PropertyInfo property = null)
          => GetInnerElementString(type, EXmlTypeTag.Summary, property);
```

Aj na iných miestach kde sa to nachádza.